ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/traffic-policies.cfg

```diff
@@ -106,13 +128,43 @@ traffic-policies
          protocol tcp
          actions
             drop
-            log
          !
       !
       match ipv4-all-default ipv4
          actions
             drop
             log
    !
+   traffic-policy BLUE-C3-POLICY
+      match ipv4-all-default ipv4
+         actions
+            count test
+            set traffic class 10
```

Comment:
Add a traffic policy where both `match ipv4-all-default ipv4` and `match ipv6-all-default ipv6` are present.